src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -260,6 +271,59 @@
       </PropertyComment>
     </Field>
   </Node>
+  <Node Name="FunctionPointerCallingConventionSyntax" Base="CSharpSyntaxNode">
+    <TypeComment>
+      <summary>Function pointer calling convention syntax.</summary>
+    </TypeComment>
+    <Kind Name="FunctionPointerCallingConvention"/>
+    <Field Name="ManagedSpecifier" Type="SyntaxToken">
```

Comment:
>ManagedSpecifier [](start = 17, length = 16)

It feels like the name implies one of the possible keywords. I think we should try to find more "neutral" name. "ManagedOrUnmanagedKeyword"? This would follow an existing pattern from "ThisOrBaseKeyword"